src/AzureAccountWrapper.ts

```diff
@@ -79,8 +79,18 @@ export class AzureAccountWrapper {
     }
 
     async query(path: string, apiVersion: string = DEFAULT_API_VERSION): Promise<any> {
+        interface PathVersion {
+            path: string;
+            version: string;
+        }
+        const customApiVersions: PathVersion[] = vscode.workspace
+            .getConfiguration("azure-resource-explorer-for-vscode")
+            .get("customApiVersions") ?? [];
+
+        const match = customApiVersions.find((x :PathVersion) => x.path === path);
+        const finalVersion = match ? match.version : apiVersion;
```

Comment:
idk how practical it'd be to have an "address" bar somewhere in the UI:
```
| [api-version] |---| [path/to/resource/] |
```
Or just someway to communicate which version it's using. Maybe in the status bar?

Reply:
We can introduce a custom logChannel, [like e.g. here](https://github.com/scale-tone/kee-shepherd/blob/c797fcef87976bf7b249eaf7bfeeceb0270dce8c/kee-shepherd-vscode/src/extension.ts#L19).

Reply:
Actually, [we do have output channel here already](https://github.com/scale-tone/azure-resource-explorer-vscode/blob/c30c81fed48ad0a676ba48796abbebabec6ca163/src/extension.ts#L10). I just forgot :)

Reply:
no worries, i found it